.changes/unreleased/Refactor-20231023-150009.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Refactor
+body: BREAKING CHANGE ownerAlias deprecated on service inputs
```

Comment:
whats the difference between this and the other change entry?  Also +1 to not being so terse - think of the end user when they are reading this and what information they need to understand what this change it and how it affects them.